### PR TITLE
update issue #1

### LINKED ISSUE #1: Update AboutUs
Each person is responsible for updating their own information.

### PR DESCRIPTION
**Pull request by dabzpengu**
added role

### CODE REVIEW

**Comment by ZHANGTIANYAO1** [comment]
LGTM

File: docs/images/gandalf.jpg
Comment (Lalelulilulela) [inline docs/images/gandalf.jpg:1]: Fabulous image

File: docs/AboutUs.md [hunk 1]
```diff
@@ -19,15 +19,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-<img src="images/johndoe.png" width="200px">
+<img src="images/gandalf.jpg" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/dabzpengu)]
 
 * Role: Team Lead
-* Responsibilities: UI
+* Responsibilities: UI / Quality Assurance
```
Comment (Lalelulilulela) [inline docs/AboutUs.md:29]: Fabulous role

**Review by Lalelulilulela — COMMENTED** [review]
Looks good

**Pull request merged**